src/models/item.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, data: dict):
             (10**item.get("price_including_taxes", {}).get("decimals", 0))
         self.price = f"{self.price:.2f}"
         self.currency = item.get("price_including_taxes", {}).get("code", "-")
+        self.next_sales_window_purchase_start = item.get("next_sales_window_purchase_start")
```

Comment:
All of the items I got in my favorites don't own the attribute `next_sales_window_purchase_start`.
What should be displayed in the notification when this information is not available? 
In this case `item.get()` will return `None` by default.
But this is not very convenient so I would suggest `NA`?

```suggestion
        self.next_sales_window_purchase_start = item.get("next_sales_window_purchase_start", "NA")
```

Can you provide an item_id of an item with this attribute so I can the implementation?